src/xcode/ENA/ENAUITests/Utils/LaunchArguments.swift

```diff
@@ -112,6 +112,25 @@ enum LaunchArguments {
 
 	}
 
+	enum familyTest {
```

Comment:
Can this be renamed? 

```suggestion
	enum familyMemberTest {
```

Reply:
yes, I'll do